lib/cast/cast_sender.js

```diff
@@ -211,7 +217,12 @@ shaka.cast.CastSender = class {
     // Use static versions of the API callbacks, since the ChromeCast API is
     // static. If we used local versions, we might end up retaining references
     // to destroyed players here.
-    const sessionRequest = new chrome.cast.SessionRequest(this.receiverAppId_);
+    const sessionRequest = new chrome.cast.SessionRequest(this.receiverAppId_,
```

Comment:
Compiler complains:
`[JSC_WRONG_ARGUMENT_COUNT] Function chrome.cast.SessionRequest: called with 4 argument(s). Function requires at least 1 argument(s) and no more than 1 argument(s).`
Even thought the cast documentation says otherwise:
[Cast Documentation - chrome.cast.SessionRequest](https://developers.google.com/cast/docs/reference/web_sender/chrome.cast.SessionRequest)